content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/magnetic_dipole_frequency/fields.rst

```diff
@@ -3,8 +3,90 @@
 Visualization of the Electromagnetic Fields
 ===========================================
 
-.. topic:: Purpose
+.. Purpose::
+
+    Here, we provide numerical modeling tools for visualizing the electric field, magnetic field and current density caused by a magnetic dipole source.
+    A set of questions about the fields and their dependency on various parameters are then presented as a learning exercise.
+    By completing this exercise, you will become comfortable with the numerical modeling tools provided and gain a fundamental understanding of the fields which are caused by a harmonic magnetic dipole.
+
```

Comment:
could you please add the link to the app on azure notebooks near the top of the page?